### ISSUE TITLE
Wrong VRAM frequency

### ISSUE THREAD

**Issue by m-rzb** [flightlessmango/MangoHud#280]
I am not sure if this is only affecting my set up. 
I have an AMD RX580 8GB which I have undervolted and overclocked the VRAM only; from 2100MHz to 2250MHz.

As you can see in the attached; when under load the VRAM frequency when overclocked is wrong -2044
Similar hwmon and freq2_input is showing -2244967296

I have compared it with Gallium_hud which is showing correct frequencies.

Let me know if you need any further information.

![ryse_mem_2250](https://user-images.githubusercontent.com/43208506/87307085-e88e2600-c510-11ea-898e-fc37b98cbd54.png)
![mclk3](https://user-images.githubusercontent.com/43208506/87307019-d1e7cf00-c510-11ea-9c4d-30ae85bea1e8.png)
![GPU_mem_clock](https://user-images.githubusercontent.com/43208506/87307023-d3b19280-c510-11ea-905f-6e3d6a759ac1.png)

**Comment by magas3000** [comment]
I have an RX 590 and I noticed the same exact issue when I overclocked my VRAM so you're not alone

**Comment by flightlessmango** [comment]
It looks like you're one or two releases behind on mangohud, best to update to make sure it's not already fixed. However it seems that the bug is with the value reported in hwmon, that's were mangohud is getting it's value from.

Just because galliumhud can get the correct value sadly doesn't mean that other apps can, galliumhud is in a rather unique situation where it's situated in the driver.

In short my guess is that it's a lm-sensors/libsensors bug and manoghud is working as intended

**Comment by m-rzb** [comment]
Thanks for reply. I am fairly certain the version I am currently using is 0.4.1, the screenshots are not new however. I still have same results, that's why I have reported it. 
I have noticed that hwmon is reporting wrong values. My thought was maybe there is a different source where mangohud could look for correct values. Unfortunately, that's beyond my nonexistent coding skills otherwise I would have done it personally. Regarding bug in libsensors I assumed that as well but could not find anything similar reported. Anyway, those wrong values are not crucial for me, just an annoyance.

**Comment by m-rzb** [comment]
The issue with [wrong values in hwmon](https://lkml.org/lkml/2020/7/7/1135) has been patched in [kernels 5.4.51 to 5.4.60](https://lkml.org/lkml/2020/7/9/303) as well as the ones after [kernel 5.7.8](https://lkml.org/lkml/2020/7/9/305). The kernels between 5.5 and 5.7.7 apparently are not patched. The kernel I was using until recently was 5.7.7:
[kernel_5 4 51](https://user-images.githubusercontent.com/43208506/91086935-895f1d80-e647-11ea-84b7-0617748ab8e7.png)
[kernel_5 7 7](https://user-images.githubusercontent.com/43208506/91086937-89f7b400-e647-11ea-98f9-f9397a172bad.png)
[kernel_5 7 8](https://user-images.githubusercontent.com/43208506/91086938-89f7b400-e647-11ea-8cee-a28dfe7382d7.png)

Maybe the information here is helpful for someone, as Ubuntu 20.04, for example, is using kernel 5.4.0 as far as I am aware and it will be still affected by this bug.

I will close the issue as solved now. 

Thanks, everyone!
